### PR TITLE
Control looping and show uuid

### PR DESCRIPTION
**Pull request by aakarshg**
Essentially this commit will facilitate:
1. showing uuid when doing a get benchmarks
2. running multiple benchmarks of even same type
3. avoid unnecessary looping

### CODE REVIEW

File: roles/ycsb-bench/templates/configmap.yml.j2 [hunk 1]
```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: ycsb-test
-  namespace: '{{ operator_namespace }}'
+  namespace: '{{ operator_namespace }}-{{ uuid }}'
```
Comment (jtaleric) [inline roles/ycsb-bench/templates/configmap.yml.j2]: bad copy/pasta?

Comment (aakarshg) [reply roles/ycsb-bench/templates/configmap.yml.j2]: yesss, nice catch will fix it :)

File: tests/common.sh [hunk 2]
```diff
@@ -37,6 +37,28 @@ function wait_clean {
...
 }
 
+# The argument is 'timeout in seconds'
+function get_uuid () {
+  counter=0
+  sleep_time=5
+  counter_max=$(( $1 / sleep_time ))
+  uuid="False"
+  until [ $uuid != "False" ] ; do
+    sleep $sleep_time
```
Comment (dry923) [inline tests/common.sh]: If I'm following this right could we move the ```sleep $sleep_time``` into your if statement? Right now we always sleep X seconds before trying the first time. If we move it we might avoid a sleep.

Comment (aakarshg) [reply tests/common.sh]: that makes sense. updated it

**Comment by jtaleric** [comment]
Is something slowing down the pulling of the Operator/Redis/Sidecar? 

@dry923 we might want to time these things -- or time all things. Jenkins has a way to present timestamps in the console (or at least I haven seen it done before). This might help us?

**Comment by dry923** [comment]
> Is something slowing down the pulling of the Operator/Redis/Sidecar?
> 
> @dry923 we might want to time these things -- or time all things. Jenkins has a way to present timestamps in the console (or at least I haven seen it done before). This might help us?

@jtaleric Agreed. I enabled timestamps in the console of Jenkins. I was thinking about other options as well (timing individual tests, etc) but we can discuss those later. 

Take a look at the output from the latest run and see if that helps any.

File: tests/common.sh [hunk 3]
```diff
@@ -37,6 +37,29 @@ function wait_clean {
...
+  sleep 5
+  counter=0
+  sleep_time=5
+  counter_max=$(( $1 / sleep_time ))
+  uuid="False"
+  until [ $uuid != "False" ] ; do
+    if [ -z $uuid ]; then
+      uuid=$(kubectl -n my-ripsaw get benchmarks -o jsonpath='{.items[0].status.uuid}')
+      sleep $sleep_time
+      uuid="False"
```
Comment (dry923) [inline tests/common.sh]: This code will never succeed. You obtain the uuid then set it to false two lines later.

Comment (jtaleric) [reply tests/common.sh]: @aakarshg

**Comment by jtaleric** [comment]
KNI built fine. Good to merge.

**Pull request merged**